library/ord-tree.lisp

```diff
@@ -357,7 +365,9 @@ Like `replace-or-insert', but prioritizing insertion as a use case."
          ((EQ) (Some (remove-leaving-double-black tre)))
          ((GT)
           (map (fn (new-right) (bubble-double-black clr left pivot new-right))
-               (remove-without-as-black right elt)))))))
+               (remove-without-as-black right elt)))))
+      (_
```

Comment:
Another place where `DoubleBlackEmpty` should be impossible, but this time we are technically inside the `remove` process, we just haven't reached a point where we would insert a double-black node yet. Possibly `(error "Encountered double-black node early in `remove` while searching for the node to remove.")`.